x-pack/legacy/plugins/apm/server/routes/security.ts

```diff
@@ -6,12 +6,12 @@
 
 import { createRoute } from './create_route';
 import { setupRequest } from '../lib/helpers/setup_request';
-import { getPermissions } from '../lib/security/getPermissions';
+import { getIndicesPrivileges } from '../lib/security/getIndicesPrivileges';
 
-export const permissionsRoute = createRoute(() => ({
-  path: '/api/apm/security/permissions',
+export const indicesPrivilegesRoute = createRoute(() => ({
+  path: '/api/apm/security/indicesPrivileges',
```

Comment:
Afaict all endpoints use camel_case. 
```suggestion
  path: '/api/apm/security/indices_privileges',
```